src/wallet/bdb.cpp

```diff
@@ -305,20 +305,20 @@ BerkeleyDatabase::~BerkeleyDatabase()
     }
 }
 
-BerkeleyBatch::BerkeleyBatch(BerkeleyDatabase& database, const char* pszMode, bool fFlushOnCloseIn) : pdb(nullptr), activeTxn(nullptr), m_cursor(nullptr), m_database(database)
+BerkeleyBatch::BerkeleyBatch(BerkeleyDatabase& database, const bool fReadOnly, bool fFlushOnCloseIn) : pdb(nullptr), activeTxn(nullptr), m_cursor(nullptr), m_database(database)
```

Comment:
nit: for new variables the naming convention is `snake_case`.

Reply:
334eadec38afd0a34edb990322fda0825118ec76

👍 and drop `this->` below.

BTW, why not just drop member `fReadOnly` since it's always `false`?

Reply:
Changed the name and dropped `this->`

> BTW, why not just drop member fReadOnly since it's always false?

I kept it, because it's set to `true` in `BerkeleyDatabase::Rewrite`.

Reply:
Where?

Reply:
https://github.com/bitcoin/bitcoin/pull/20130/files#diff-b06753f6e4621f5bcd6cd27fed4137e70a262596b08f7f3a1416ea21977c53d4R467